Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -1035,8 +1035,10 @@ static void setSyncOnClock(GstElement *element, bool sync)
 
 void MediaPlayerPrivateGStreamer::syncOnClock(bool sync)
 {
+#if !USE(WESTEROS_SINK)
```

Comment:
Why is non-sync preferred on Westeros sink? In general, sync is preferred on all the sinks, precisely to avoid GStreamer processing faster than realtime and doing unneeded work. Some clarification in the commit message would be great.

Reply:
WesterosSink incapsulates decoder and does A/V sync outside of GStreamer on most configurations. Enforcing sink to sync on Gst clock causes playback stuttering(because it effectively throttles samples enqueuing video data to HW). On configuration where sync on GST clock is desired, it can still set it by itself if required. 

From git history, I understood that the original 'syncOnClock' change was intended to disable sync for WebRTC cases. For this use case, WesterosSink iface provides another configuration property. This configuration will come in separate PR.

Reply:
Ah, ok, yes. If the decoding is done inside the sink, it's not behaving as a regular GStreamer sink and it makes sense to disable syncOnClock.